docs/release-notes/release-notes-0.18.0.md

```diff
@@ -185,6 +185,14 @@
   [MinConf](https://github.com/lightningnetwork/lnd/pull/8097)(minimum number
   of confirmations) has been added to the `WalletBalance` RPC call.
 
+* Previously when callng `SendCoins`, `SendMany`, `OpenChannel` and
+  `CloseChannel` for coop close, it is allowed to specify both an empty
```

Comment:
Good area to start using an `fn.Either` as mentioned before to encode this restriction into the types themselves.

Reply:
Looks like the `tlv` package is not updated in go mod yet, will do and see how it can help.

Reply:
Seems like this PR did not touch any of these functions, currently `SatPerVbyte` and `TargetConf` are still both optional fields at least in `sendCoins`, would there be upcoming PRs to  enforce this from the client side?

Reply:
`SendCoins` use `CalculateFeeRate` which enforce this check.

Reply:
Yeah so I think there should be a check [here](https://github.com/lightningnetwork/lnd/blob/2b74a34301fe4a90b2b4f02b075bc93c938fe410/cmd/lncli/commands.go#L339-L452) on the lncli side to ensure at least either targetconf or satvbyte is set. Maybe for OpenChannel, CloseChannel etc. as well.

Reply:
Updating the docs on flags as well as they are both optional.

I do not have much context about this change or why it is needed but we can either ensure these lncli commands have these flags or if they are missing the SendCoins RPC function and other functions  can set a default before it is passed to the `CalculateFeeRate` function, for user experience.

Reply:
think an error will be returned from the cli if not set since it's calling the RPC? Don't think it's a good idea to set the default here - it's a very sensitive param, and the users need to set it themselves.